docs/nodes/mamaki.md

```diff
@@ -32,41 +55,43 @@ git clone https://github.com/celestiaorg/networks.git
 ```
 
 To initialize the network pick a "node-name" that describes your
-node. The --chain-id parameter we are using here is `devnet-2`. Keep in
+node. The --chain-id parameter we are using here is `mamaki`. Keep in
 mind that this might change if a new testnet is deployed.
 
 ```sh
-celestia-appd init "node-name" --chain-id devnet-2
+celestia-appd init "node-name" --chain-id mamaki
 ```
 
-Copy the `genesis.json` file. For devnet-2 we are using:
+Copy the `genesis.json` file. For mamaki we are using:
 
 ```sh
-cp $HOME/networks/devnet-2/genesis.json $HOME/.celestia-app/config
+cp $HOME/networks/mamaki/genesis.json $HOME/.celestia-app/config
 ```
 
 Set seeds and peers:
 
 ```sh
-SEEDS="74c0c793db07edd9b9ec17b076cea1a02dca511f@46.101.28.34:26656"
-PEERS="34d4bfec8998a8fac6393a14c5ae151cf6a5762f@194.163.191.41:26656"
+SEEDS="f0c58d904dec824605ac36114db28f1bf84f6ea3@144.76.112.238:26656"
+PEERS="e4429e99609c8c009969b0eb73c973bff33712f9@141.94.73.39:43656"
 sed -i.bak -e "s/^seeds *=.*/seeds = \"$SEEDS\"/; s/^persistent_peers \
     *=.*/persistent_peers = \"$PEERS\"/" $HOME/.celestia-app/config/config.toml
 ```
 
+Note: You can find more peers [here](https://github.com/celestiaorg/networks/blob/master/mamaki/peers.txt).
+
 You can return back to where you left off in the Bridge Node guide [here](../nodes/validator-node.md#configure-pruning)
 
 ## Quick-Sync With Snapshot
 
 Run the following command to quick-sync from a snapshot for
-`devnet-2` (Note: this is a 100 GB download):
+`mamaki` (Note: this is a 100 GB download):
```

Comment:
@YazzyYaz It's not a 100GB download, especially now with the new chain it will be much lower than that in the early days (1GB atm). As an example, the last snapshot of devnet-2 is 80GB (after 5 months of data).